open-apparel-registry/open-apparel-registry
# Update Reject Button Text

## Overview

Dashboard activity report reject dialog has a confirm button. The
button has been updated to display the text, "Reject".

Connects #1241 

## Demo

<img width="370" alt="Screen Shot 2021-02-16 at 12 09 06 PM" src="https://user-images.githubusercontent.com/21046714/108097186-d1d42280-704f-11eb-8069-46f9f3099f94.png">

## Testing Instructions

* Run `./scripts/server`
* Submit a closure activity report
* Log in as an admin and browse [activity reports](http://localhost:6543/dashboard/activityreports)
* Click reject on the pending activity report.
    * The buttons should say 'cancel' and 'reject'

## Checklist
